From 344d755e78e7600bfd3bea39b83c7d3584aa325f Mon Sep 17 00:00:00 2001
From: Ludwig Krispenz <lkrispen@redhat.com>
Date: Tue, 30 Jul 2013 11:29:47 +0200
Subject: [PATCH 96/99]  fix compiler warning (cherry picked from commit
 904416f4631d842a105851b4a9931ae17822a107) (cherry picked from commit
 3a5f8de21fba3656670b8ee35e020f159d4110db) (cherry picked from commit
 efcb1af30b63f91d3235914962bd4aa399bc08ba) (cherry picked from commit
 d9a1c7b527d7252195fac15636c22101ceb775bf)

---
 ldap/servers/plugins/acl/aclutil.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/ldap/servers/plugins/acl/aclutil.c b/ldap/servers/plugins/acl/aclutil.c
index 2002276..4f81bc0 100644
--- a/ldap/servers/plugins/acl/aclutil.c
+++ b/ldap/servers/plugins/acl/aclutil.c
@@ -1400,7 +1400,6 @@ void acl_ht_add_and_freeOld(acl_ht_t * acl_ht,
 void acl_ht_remove_and_free(acl_ht_t * acl_ht,
 					PLHashNumber key){
 	char *old_value = NULL;	
-	uintptr_t pkey = (uintptr_t)key;
 
 	if ( (old_value = (char *)acl_ht_lookup( acl_ht, key)) != NULL ) {
 		acl_ht_remove( acl_ht, key);
-- 
1.8.1.4

